<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Comm_Window</class>
 <widget class="QMainWindow" name="Comm_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>824</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Comm_Window</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>8</y>
      <width>961</width>
      <height>741</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Comm_Console" stretch="2,10,3">
     <item>
      <layout class="QVBoxLayout" name="TitleandSerialLayout" stretch="2,1">
       <item>
        <widget class="QLabel" name="WheelBotComm">
         <property name="font">
          <font>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Communication System Diagnostic</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Serial_Layout">
         <property name="spacing">
          <number>9</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Port:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="PortcomboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Connect_Btn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Disconnect_Btn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Disconnect</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="Lo_Plots"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ControlsandData">
       <item>
        <layout class="QVBoxLayout" name="GainsPanel">
         <item>
          <widget class="QLabel" name="PId_Loop_Gain">
           <property name="text">
            <string>Control Signal Slider</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="AmplitudeValue">
           <property name="font">
            <font>
             <pointsize>17</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>0.0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="K_pLayout">
           <item>
            <widget class="QSlider" name="AmplitudeSlider">
             <property name="sliderPosition">
              <number>50</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
             <property name="tickInterval">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="MotSpd_DatLayout">
         <item>
          <widget class="QLabel" name="Rotat_PWM">
           <property name="text">
            <string>TBD</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Motor_Speed">
           <property name="text">
            <string>TBD</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="MotorSpdLayout">
           <item>
            <widget class="QLabel" name="LMotLbl">
             <property name="text">
              <string>tbd:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Lft_Motor_Speed">
             <property name="text">
              <string>tbd</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="RMotLbl">
             <property name="text">
              <string>tbd:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Rgt_Motor_Speed">
             <property name="text">
              <string>tbd</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="Load_Data">
           <property name="text">
            <string>TBD</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="LoadDat_Layout">
           <item>
            <widget class="QLineEdit" name="LoadLineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="Load_Btn">
             <property name="text">
              <string>TBD</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="RcdandPlybckLo">
           <item>
            <widget class="QPushButton" name="Record_Btn">
             <property name="text">
              <string>TBD</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Playback_Btn">
             <property name="text">
              <string>TBD</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
